.github/workflows/ci.yml

```diff
@@ -100,14 +100,33 @@ jobs:
       # Install theme and dependencies
       - run: |
           pip install .
-          pip install mkdocs-minify-plugin>=0.2
+          pip install mkdocs-minify-plugin>=0.2    
 
       # Build documentation
       - env:
           GOOGLE_ANALYTICS_KEY: ${{ secrets.GOOGLE_ANALYTICS_KEY }}
         run: |
           mkdocs gh-deploy --force
           mkdocs --version
+          
+      - name: Waiting for 200 from the Netlify Preview
```

Comment:
Note that it's currently not run for this PR, as the deploy job does solely run on `master`. Thus, it makes no sense to have it here. Also, this will probably increase build time.

Reply:
you can specify how long it waits. and yeah well that depends on where you would want it to run.